.github/workflows/ubuntu_clean_llvm_build.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       with:
         mode: rebuild
     - if: env.rebuild == '1'
-      uses: actions/setup-python@v1
```

Comment:
I think the Line 44 is the one should be changed.  `actions/upload-artifact@v3`

Reply:
Fixed ;)